package.json

```diff
@@ -38,6 +39,7 @@
     "got": "^13.0.0",
     "lqip-modern": "^2.0.0",
     "lucide-react": "^0.292.0",
+    "nanoid": "^5.0.3",
```

Comment:
**question:** why do we need this dependency?

Reply:
Oh.. I forgot about this.